pkg/pgmodel/migrations/sql/preinstall/005-install_uda.sql

```diff
@@ -0,0 +1,25 @@
+CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.get_timescale_major_version()
+    RETURNS INT
+AS $func$
+    SELECT split_part(extversion, '.', 1)::INT FROM pg_catalog.pg_extension WHERE extname='timescaledb' LIMIT 1;
+$func$
+LANGUAGE SQL STABLE PARALLEL SAFE;
+
+--just a stub will be replaced in the idempotent scripts
+CREATE OR REPLACE PROCEDURE SCHEMA_CATALOG.execute_maintenance_job(job_id int, config jsonb)
+AS $$
+BEGIN
+    RAISE 'calling execute_maintenance_job stub, should have been replaced';
+END
+$$ LANGUAGE PLPGSQL;
+
+
+--add 2 jobs executing every 30 min by default for timescaledb 2.0
+DO $$
+BEGIN
+    IF SCHEMA_CATALOG.get_timescale_major_version() >= 2 THEN
+       PERFORM add_job('SCHEMA_CATALOG.execute_maintenance_job', '30 min');
+       PERFORM add_job('SCHEMA_CATALOG.execute_maintenance_job', '30 min');
+    END IF;
+END
+$$;
```

Comment:
should we add a check that there's at least 1 job to the `idempotent` scripts? I'm concercned about how we'll handle the eventual upgrades from Timescale 1 to 2

Reply:
I dunno what to do about that. Having 0 jobs is totally reasonable if using cron. We'll need some manual intervention.